tests/cases/fourslash/unusedImports13FS.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path='fourslash.ts' />

```

Comment:
It's the 13th in a series of files that have that exact naming convention.... I think FS stands for fourslash.